src/Microsoft.Health.Fhir.Core/Features/Search/SearchResult.cs

```diff
@@ -68,5 +70,10 @@ public SearchResult(int totalCount, IReadOnlyList<Tuple<string, string>> unsuppo
         /// Gets the continuation token.
         /// </summary>
         public string ContinuationToken { get; }
+
+        /// <summary>
+        /// Gets if this result is partia;.
```

Comment:
typo. Also we should probably make it clear that this is not related to the continuation token